modules/animate/main/index.coffee

```diff
@@ -1,8 +1,13 @@
-### istanbul ignore next: ignore coffeescript generated code that can't be covered  ###
+transition = require('modules/transition/main')
+EventEmitter = require('modules/event-emitter/main')
+interpolate = require('modules/interpolate/main')
+select = require('modules/selection/main')
+hxMap = require('modules/map/main')
```

Comment:
There shouldn't be a global map, so it could be renamed to just map. I kind of wanted it to be `Map` rather than `map` so its clear it's a type that needs creating with new, but that would also be confusing since it isn't really a Map. Maybe HMap would be a bit clearer?